src/ports/postgres/modules/convex/test/mlp.sql_in

```diff
@@ -0,0 +1,832 @@
+{1,0.11132,0.00,27.740,0,0.6090,5.9830,83.50,2.1099,4,711.0,20.10,396.90,13.35} | 2 | 20.10
+{1,0.17331,0.00,9.690,0,0.5850,5.7070,54.00,2.3817,6,391.0,19.20,396.90,12.01} | 2 | 21.80
+{1,0.27957,0.00,9.690,0,0.5850,5.9260,42.60,2.3817,6,391.0,19.20,396.90,13.59} | 2 | 24.50
+{1,0.17899,0.00,9.690,0,0.5850,5.6700,28.80,2.7986,6,391.0,19.20,393.29,17.60} | 2 | 23.10
+{1,0.28960,0.00,9.690,0,0.5850,5.3900,72.90,2.7986,6,391.0,19.20,396.90,21.14} | 2 | 19.70
+{1,0.26838,0.00,9.690,0,0.5850,5.7940,70.60,2.8927,6,391.0,19.20,396.90,14.10} | 2 | 18.30
+{1,0.23912,0.00,9.690,0,0.5850,6.0190,65.30,2.4091,6,391.0,19.20,396.90,12.92} | 2 | 21.20
+{1,0.17783,0.00,9.690,0,0.5850,5.5690,73.50,2.3999,6,391.0,19.20,395.77,15.10} | 2 | 17.50
+{1,0.22438,0.00,9.690,0,0.5850,6.0270,79.70,2.4982,6,391.0,19.20,396.90,14.33} | 2 | 16.80
+{1,0.06263,0.00,11.930,0,0.5730,6.5930,69.10,2.4786,1,273.0,21.00,391.99,9.67} | 2 | 22.40
+{1,0.04527,0.00,11.930,0,0.5730,6.1200,76.70,2.2875,1,273.0,21.00,396.90,9.08} | 2 | 20.60
+{1,0.06076,0.00,11.930,0,0.5730,6.9760,91.00,2.1675,1,273.0,21.00,396.90,5.64} | 2 | 23.90
+{1,0.10959,0.00,11.930,0,0.5730,6.7940,89.30,2.3889,1,273.0,21.00,393.45,6.48} | 2 | 22.00
+{1,0.04741,0.00,11.930,0,0.5730,6.0300,80.80,2.5050,1,273.0,21.00,396.90,7.88} | 2 | 11.90
+\.
+
+-- Normalize the columns
+CREATE TEMPORARY TABLE maxs as(
+    SELECT
+    max(x[1]) m1,
+    max(x[2]) m2,
+    max(x[3]) m3,
+    max(x[4]) m4,
+    max(x[5]) m5,
+    max(x[6]) m6,
+    max(x[7]) m7,
+    max(x[8]) m8,
+    max(x[9]) m9,
+    max(x[10]) m10,
+    max(x[11]) m11,
+    max(x[12]) m12,
+    max(x[13]) m13,
+    max(x[14]) m14
+    from lin_housing_wi
+);
+
+CREATE TABLE lin_housing_wi_scaled AS
+SELECT ARRAY[
+    x[1]/(SELECT m1 from maxs),
+    x[2]/(SELECT m2 from maxs),
+    x[3]/(SELECT m3 from maxs),
+    x[4]/(SELECT m4 from maxs),
+    x[5]/(SELECT m5 from maxs),
+    x[6]/(SELECT m6 from maxs),
+    x[7]/(SELECT m7 from maxs),
+    x[8]/(SELECT m8 from maxs),
+    x[9]/(SELECT m9 from maxs),
+    x[10]/(SELECT m10 from maxs),
+    x[11]/(SELECT m11 from maxs),
+    x[12]/(SELECT m12 from maxs),
+    x[13]/(SELECT m13 from maxs),
+    x[14]/(SELECT m14 from maxs)] as x,
+    id,y
+FROM lin_housing_wi;
+
+DROP TABLE IF EXISTS maxs;
+DROP TABLE IF EXISTS mlp_regress;
+
+
+-- Convergence varies for regression
+
+SELECT setseed(0);
+SELECT mlp_regression(
+    'lin_housing_wi_scaled',           -- Source table
+    'mlp_regress',              -- Desination table
+    'x',                        -- Input features
+    'y',                        -- Dependent variable
+    ARRAY[5,5],                 -- Number of units per layer
+    'step_size=0.007,
+    n_iterations=300,
+    tolerance=0',
+    'sigmoid');
+
+
+SELECT assert(
+    loss < 5000, -- Corresponds to RMSE of sqrt(5000/506) = 3.14.
+    'MLP: Loss is too high (> 0.1). Wrong result.'
+) FROM mlp_regress;
+
+DROP TABLE IF EXISTS lin_housing_wi_scaled;
```

Comment:
Might it make sense to cleanup other resources (tables) created/used during the test run?  I manually ran the mlp tests and noticed the following resources are not removed.

```
madlibtest=# \d
                  List of relations
 Schema |         Name          |   Type   |  Owner
--------+-----------------------+----------+---------
 public | iris_data             | table    | eespino
 public | lin_housing_wi        | table    | eespino
 public | lin_housing_wi_id_seq | sequence | eespino
 public | mlp_class             | table    | eespino
 public | mlp_class_summary     | table    | eespino
 public | mlp_regress           | table    | eespino
 public | mlp_regress_summary   | table    | eespino
(7 rows)
```